cs/shipdesign.go

```diff
@@ -348,9 +348,9 @@ func ComputeShipDesignSpec(rules *Rules, techLevels TechLevel, raceSpec RaceSpec
 				torpedoJammersByCount[component.TorpedoJamming] += slot.Quantity
 			}
 
-			// beam bonuses
-			spec.BeamBonus += component.BeamBonus * float64(slot.Quantity)
-			spec.BeamDefense += component.BeamDefense * float64(slot.Quantity)
+			// beam bonuses (capacitors are capped at 2.55x; beam deflectors are unlimited)
+			spec.BeamBonus += math.Min(math.Pow(1+component.BeamBonus, float64(slot.Quantity))-1, 1.55)
```

Comment:
shouldn't this be 2.55 not 1.55?

Reply:
IIRC the beam bonus gets added on to the beam's power (so the total multi is 1+beamBonus), so 2.55 would actually result in a total boost of 3.55x.